java/ql/lib/semmle/code/java/dataflow/DataFlow.qll

```diff
@@ -6,6 +6,10 @@
 import java
 
 module DataFlow {
-  import semmle.code.java.dataflow.internal.DataFlowImpl
+  import semmle.code.java.dataflow.internal.DataFlow
   import semmle.code.java.dataflow.internal.DataFlowImpl1
+
+  module Internal {
```

Comment:
I think it would be better not to have this, and let uses of this go directly to `semmle.code.java.dataflow.internal.DataFlowImpl` instead.